ci/test/06_script_a.sh

```diff
@@ -20,6 +20,26 @@ END_FOLD
 DOCKER_EXEC mkdir -p "${BASE_BUILD_DIR}"
 export P_CI_DIR="${BASE_BUILD_DIR}"
 
+BEGIN_FOLD apk
+if [ -n "$ANDROID_TOOLS_URL" ]; then
+  DOCKER_EXEC "${BASE_ROOT_DIR}/configure" --prefix=$DEPENDS_DIR/armv7a-linux-android || ( (DOCKER_EXEC cat config.log) && false)
+  DOCKER_EXEC make $MAKEJOBS $GOAL || ( echo "Build failure. Verbose build follows." && DOCKER_EXEC make $GOAL V=1 ; false )
+  DOCKER_EXEC make -C src/qt apk
+
+  DOCKER_EXEC "${BASE_ROOT_DIR}/configure" --prefix=$DEPENDS_DIR/aarch64-linux-android || ( (DOCKER_EXEC cat config.log) && false)
+  DOCKER_EXEC make $MAKEJOBS $GOAL || ( echo "Build failure. Verbose build follows." && DOCKER_EXEC make $GOAL V=1 ; false )
+  DOCKER_EXEC make -C src/qt apk
+
+  DOCKER_EXEC "${BASE_ROOT_DIR}/configure" --prefix=$DEPENDS_DIR/i686-linux-android || ( (DOCKER_EXEC cat config.log) && false)
+  DOCKER_EXEC make $MAKEJOBS $GOAL || ( echo "Build failure. Verbose build follows." && DOCKER_EXEC make $GOAL V=1 ; false )
+  DOCKER_EXEC make -C src/qt apk
+
+  DOCKER_EXEC "${BASE_ROOT_DIR}/configure" --prefix=$DEPENDS_DIR/x86_64-linux-android || ( (DOCKER_EXEC cat config.log) && false)
```

Comment:
It is probably enough to keep only the `aarch64-linux-android` build.

Reply:
Agreed. Okay to just comment the others out in case somebody wants them?

Reply:
I think it would be easy to re-create them by copy-paste and adjusting the name, etc.